.github/workflows/ci.yml

```diff
@@ -146,7 +146,7 @@ jobs:
     needs:
       - pre_commit
     if: >
-      github.event_name == 'push' &&
+      (github.event_name == 'push' || github.actor == 'dependabot[bot]') &&
```

Comment:
Maybe we should simply reconsider having `github.event_name == 'push'` in the first place.

Reply:
I'm also in favor of running the PR builds. I think that Tom's intention was to limit the build time by disabling duplicated tests, but IMO PR build are more valuable then push ones. We should switch to PR builds, or at least re-enable them 🤔  WDYT @mpichura ?

Reply:
You know I'm in favour of running heavy tests on PR build rather than push. I just don't want to make such changes while Tom's on PTO.

Reply:
Ok, so let merge it as it is and return to this topic after Tom's return from PTO. Do you agree @dsibilio ?

Reply:
Sure, go ahead